cmd/demo/node_setup.go

```diff
@@ -109,63 +109,74 @@ func (n *node) setup() error {
 	return nil
 }
 
-// setupContracts reads from the config file whether the node should deploy or use
-// existing contract addresses.
-func (n *node) setupContracts() (err error) {
-	var adjAddr, assAddr common.Address
+func newTransactionContext() (context.Context, context.CancelFunc) {
```

Comment:
Please reorder the functions such that they are in order of importance.  eg:  
- setup
- setupPersistence
- setupContracts
- validateAdjudicator
- validateAssetHolder
- deployAdjudicator
- deployAssetHolder
- newTransactionContext

Reply:
ok, but changed order of setup persistence and setup contracts as setup contracts is called first.